### DISCUSSION TITLE
Incorrect Information in the Documentation

### DISCUSSION THREAD

**Discussion by uskovgs** [quarto-dev/quarto-cli#5948] [Feature Requests]
### Description

Hi!

I wanted to bring to your attention an error I noticed on your website, specifically in [Table 2](https://quarto.org/docs/authoring/tables.html#tbl-planet-measures) of the documentation.
The Sun and the Moon are not classified as planets. The Sun is a star, while the Moon is Earth's natural satellite. Additionally, I have observed that the masses listed in the table are also incorrect.

Errors have been fixed in the code below.

```python
#| label: tbl-planet-measures
#| tbl-cap: Planets

from IPython.display import Markdown
from tabulate import tabulate
table = [["Sun",696000,1.989e30],
         ["Earth",6371,5.972e24],
         ["Moon",1737,7.34e22],
         ["Mars",3390,6.39e23]]
Markdown(tabulate(
  table, 
  headers=["Astronomical object","R (km)", "mass (kg)"]
))
```

**Comment by mcanouil** [comment]
I am closing this here in favour of the issue #5952.
Thanks for letting us know.
